docs/baseline-protocol-code/branch-naming.md

```diff
@@ -0,0 +1,27 @@
+# Branch Naming Standard
+
+## Main
+
+*main* - main development branch, feature, bugfix and release branches branched from it. Changes only through the PR process.
+
+## Feature
+
+*feature/XYZ-this-is-a-new-feature-branch* - related issue XYZ.
```

Comment:
XYZ is the corresponding issue number? Can we add something to make that clear. like " related issue number XYZ"

Reply:
added